$ OpenBSD: $

- ignore bundled abseil-cpp, openh264, usrsctp, vpx to prefer our ports
- disable unused and unported linux specific pipewire code

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -63,13 +63,9 @@ include(cmake/init_target.cmake)
 include(cmake/generate_target.cmake)
 include(cmake/target_yasm_sources.cmake)
 include(cmake/external.cmake)
-include(cmake/libabsl.cmake)
-include(cmake/libopenh264.cmake)
 include(cmake/libpffft.cmake)
 include(cmake/librnnoise.cmake)
 include(cmake/libsrtp.cmake)
-include(cmake/libusrsctp.cmake)
-include(cmake/libvpx.cmake)
 include(cmake/libyuv.cmake)
 if (APPLE)
     include(cmake/libsdkmacos.cmake)
@@ -2077,8 +2073,6 @@ PRIVATE
     modules/desktop_capture/window_capturer_null.cc
 
     # linux specific
-    modules/desktop_capture/linux/base_capturer_pipewire.cc
-    modules/desktop_capture/linux/base_capturer_pipewire.h
     modules/desktop_capture/linux/mouse_cursor_monitor_x11.cc
     modules/desktop_capture/linux/mouse_cursor_monitor_x11.h
     modules/desktop_capture/linux/screen_capturer_x11.cc
@@ -2370,16 +2364,19 @@ if (APPLE)
     )
 endif()
 
-set(vpx_export)
-if (is_x86 OR is_x64)
-    set(vpx_export
-        libvpx_mmx
-        libvpx_sse2
-        libvpx_ssse3
-        libvpx_sse4
-        libvpx_avx
-        libvpx_avx2
-    )
+if (NOT VPX_FOUND)
+    set(vpx_export libvpx)
+    if (is_x86 OR is_x64)
+        set(vpx_export
+            libvpx
+            libvpx_mmx
+            libvpx_sse2
+            libvpx_ssse3
+            libvpx_sse4
+            libvpx_avx
+            libvpx_avx2
+        )
+    endif()
 endif()
 
 set(tg_owt_export tg_owt)
